src/WalkingModule/app/robots/iCubGenova04/dcm_walking_iFeel_joint_retargeting.ini

```
@@ -0,0 +1,61 @@
+# Remove this line if you don't want to use the MPC
+# use_mpc                            1
+
+# Remove this line if you don't want to use the QP-IK
+use_QP-IK                          1
+
+# Remove this line if you don't want to use osqp to
+# solve QP-IK. In this case qpOASES will be used
+# use_osqp                           1
+
+# remove this line if you don't want to save data of the experiment
+#dump_data                          1
+
+# general parameters
+[GENERAL]
+name                    walking-coordinator
+# height of the com
+com_height              0.53
+# sampling time
+sampling_time           0.01
+# enable joint retargeting
+use_joint_retargeting    1
+# enable the virtualizer
+use_virtualizer         1
+use_com_retargeting     1
```

I think this needs to be zero

---

it depends if iFeel provides the CoM position. @kouroshD, did you disable the port for iFeel?

If I'm not mistaken, you (@S-Dafarra ) told me that with this flag on the walking was failing at startup. If yes, I've no problem apply the following suggestion
```suggestion
use_com_retargeting     0
```

---

Yes, it was failing. I did not investigate too deeply who was supposed to open that port.